apps/src/lib/levelbuilder/code-docs-editor/AllCodeDocs.jsx

```diff
@@ -0,0 +1,104 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import ProgrammingEnvironmentsTable from './ProgrammingEnvironmentsTable';
+import ProgrammingExpressionsTable from './ProgrammingExpressionsTable';
+import {TextLink} from '@dsco_/link';
+
+export default function AllCodeDocs({programmingEnvironments, allCategories}) {
```

Comment:
(minor) If you end up doing another rev, a header comment describing this component might be helpful.  (No need to do this if there are no other changes.)